### DISCUSSION TITLE
Post upgrade to 2.1.4 tracking domain broken

### DISCUSSION THREAD

**Discussion by alex-pommier** [postalserver/postal#2499] [Help with using Postal]
My tracking domain on postal 2.1.1 was working correctly and i've recently upgraded to 2.1.4 and my tracking domain now throws an error on any link in emails to the below:

```
Access to click.mydomain.com was denied
You don't have authorization to view this page.
HTTP ERROR 403
``` 

My CaddyFile looks correct and is unchanged from the working 2.1.1 version
(NOTE) I have two domains in my caddy file corresponding to 2 email servers in this instance of Postal SMTP.
```
# Update this to match the hostname you wish to use for accessing the
# Postal web interface.
postal.mydomain.com {
  reverse_proxy 127.0.0.1:5000
}

# If you use open/click tracking you may need to add additional hosts
# to this file using the format below.

 click.mydomain.com {
   reverse_proxy 127.0.0.1:5000 {
     header_up X-Postal-Track-Host "1"
   }
 }
 click.mydomain2.com {
   reverse_proxy 127.0.0.1:5000 {
     header_up X-Postal-Track-Host "1"
   }
 }
``` 

Also checked my DNS that is still pointing to the correct IP and nothing has changed on my DNS.

**Comment by willpower232** [comment]
Can you share a screenshot of that error message? I'm not sure that error comes from Postal

**Reply by alex-pommier** [reply]
![Desktop-screenshot](https://github.com/postalserver/postal/assets/13565738/783453cd-89e9-4726-8c1e-174c31a990df)

**Reply by alex-pommier** [reply]
That shows up after clicking any link in an email through the postal interface. The redirect to the correct link doesn't happen.

**Reply by willpower232** [reply]
wonder if this is related to #2476 / #2415

**Reply by alex-pommier** [reply]
@willpower232 Yes I think that's the same issue I'm experiencing. I posted on that thread, thank you.
